.github/workflows/ci.yml

```diff
@@ -86,12 +86,12 @@ jobs:
           func: true
           nistkat: true
           kat: true
-      - name: native tests (+debug)
+      - name: native tests (+debug+memsan+ubsan)
         uses: ./.github/actions/multi-functest
         with:
           gh_token: ${{ secrets.GITHUB_TOKEN }}
           compile_mode: native
-          cflags: "-DMLKEM_DEBUG"
+          cflags: "-DMLKEM_DEBUG -fsanitize=address -fsanitize=undefined"
```

Comment:
Did you test that this actually fails in CI?
We have been additionally using `-fno-sanitize-recover=all` before.

Reply:
Done